content/blog/2019-11-scales-1-1-0/index.Rmarkdown

```diff
@@ -0,0 +1,118 @@
+---
+title: scales 1.1.0
+author: Hadley Wickham
+date: '2019-11-12'
+slug: scales-1-1-0
+categories:
+  - package
+tags:
+  - ggplot2
+  - scales
+  - r-lib
+photo: 
+  url: https://unsplash.com/photos/d0CasEMHDQs
+  author: David Clode
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>", 
+  fig.width = 5, 
+  fig.align = 'center',
+  fig.asp = 0.25, # 1 / phi
+  out.width = "700px",
+  fig.path = "figs/"
+)
+```
+
+We're thrilled to announce the release of [scales](https://scales.r-lib.org/) 1.1.0.
+The scales package provides much of infrastructure the underlies ggplot2's scales, and using it allow you to customize the transformations, breaks, and labels used by ggplot2. Get the latest version with:
```

Comment:
```suggestion
The scales package provides much of the infrastructure the underlies ggplot2's scales, and using it allow you to customize the transformations, breaks, and labels used by ggplot2. Get the latest version with:
```

Reply:
> much of the infrastructure *that* underlies

Don't know how to suggest changes to suggestions